ill.js

```diff
@@ -0,0 +1,35 @@
+//ryanve.dev/varam inspo
```

Comment:
@jrdnrch [i abstracted my so called varam technique](https://ryanve.dev/varam)

Reply:
I'm diggin' this man!